### PR TITLE
David-Dev

### PR DESCRIPTION
**Pull request by dhurley-suncor**
Addressed comments in SQL query from push to Ahmed-Dev. Getting on correct branch now.

### CODE REVIEW

File: queries/cellPolyRecursion.sql [hunk 1]
```diff
@@ -0,0 +1,58 @@
+/*
```
Comment (dhurley-suncor) [inline queries/cellPolyRecursion.sql:1]: is there a way I can save this as a function and import into the cell_boundary.sql file to format the gps points? This way I can separate the logic and add readability?

File: queries/get_dozer_passes_in_cell_boundary.sql [hunk 2]
```diff
@@ -0,0 +1,25 @@
+/*
+ * Creator: David
+ * Date Created: June 04, 2021
+ * Description: Get dozer coordinates between start and end pour dates within specific cell boundary, here using cellid 12_01_PIER
+ */
+
+DECLARE @sounding_cell_start_pour AS DATE = '2021-02-19' -- the start pour date of the CP sounding of interest
+DECLARE @sounding_cell_end_pour AS DATE = '2021-03-12' -- the end pour date of the CP sounding of interest
+DECLARE @cell_boundary_lowX AS INT = 149601
```
Comment (dhurley-suncor) [inline queries/get_dozer_passes_in_cell_boundary.sql:9]: The results of get_cell_boundary.sql is a csv with the boundary points, ideally I would import the lowest X/Y and highest X/Y from this file to here, easy way to do this?

File: queries/get_dozer_passes_in_cell_boundary.sql [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+FROM (
+    SELECT * FROM GPSArchive.GPS.LpPath_2021_02
+    UNION ALL 
+    SELECT * FROM GPSArchive.GPS.LpPath_2021_03
+) as Dozer_Paths_Feb_Mar_2021
+WHERE Dozer_Paths_Feb_Mar_2021.EqmtID LIKE 'T%' -- dozers always start with Txxxx
+    AND Dozer_Paths_Feb_Mar_2021.Velocity > 0
+    AND Dozer_Paths_Feb_Mar_2021.Time BETWEEN @sounding_cell_start_pour AND @sounding_cell_end_pour
+    AND Dozer_Paths_Feb_Mar_2021.X BETWEEN @cell_boundary_lowX AND @cell_boundary_highX -- only want dozer gps coordinates for pings that occur within approximatley the cell_id
+    AND Dozer_Paths_Feb_Mar_2021.Y BETWEEN @cell_boundary_lowY AND @cell_boundary_highY
```
Comment (dhurley-suncor) [inline queries/get_dozer_passes_in_cell_boundary.sql:25]: Right now I run the query and then export manually, is this best or is it better to have SQL export as CSV in the code?

File: queries/get_dozer_passes_in_cell_boundary.sql [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+DECLARE @sounding_cell_end_pour AS DATE = '2021-03-12' -- the end pour date of the CP sounding of interest
+DECLARE @cell_boundary_lowX AS INT = 149601
+DECLARE @cell_boundary_highX AS INT = 150349
+DECLARE @cell_boundary_lowY AS INT = 247960
+DECLARE @cell_boundary_highY AS INT = 248332
+
+
+SELECT EqmtID, Time, X, Y, Velocity
+FROM (
+    SELECT * FROM GPSArchive.GPS.LpPath_2021_02
```
Comment (dhurley-suncor) [inline queries/get_dozer_passes_in_cell_boundary.sql:17]: Ideally we would use the declared start and end pour dates above to reference the Table, easy way to do this?

File: .vscode/settings.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (ahasan-suncor) [inline .vscode/settings.json:1]: add .vs/ to the .gitignore file

**Review by ahasan-suncor — APPROVED** [review]
Everything looks good. All those sql stuff we can automate it later.

**Pull request merged**